### PR TITLE
Leon read me

### PR DESCRIPTION
**Pull request by rusticlion**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Draft reviewed. Further review and revision to proceed as development does.

# Checklist:

- [ ] My code has been reviewed by at least one peer.

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+**STRONGR** is a workout journal for the connected age. Schedule future workouts, review past workouts, and even bring STRONGR into the gym with you to track your performance on every exercise in your routine.
+
+# DISCLAIMER - STRONGR IS UNDER ACTIVE DEVELOPMENT AT LAMBDA LABS. THIS REPO IS NOT FEATURE COMPLETE.
+
+## Building Locally
+  The repo contains two main directories: `server` and `client`. You will need to build each to run the app for local development. Commands for each directory assume you start at the route of that directory.
+  
+  Server:
+```npm install && nodemon server.js```
```
Comment (phillipsdev) [inline ReadMe.md:11]: Actually for the nodemon you can just `npm start` as well 👍

File: ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  * `/login`
+    * { username, password }
+    * Searches for a User document and responds with its token if one is found.
+
+  * `/auto-login`
+    * No request body, but route requires a valid JWT in the "x-access-token" header.
+    * Checks a provided token for validity, responding with the token and an OK status if the token is good. For an example of usage, see the `AccessControl` component.
+
+  * `/forgot_password`
+    * { username, email }
```
Comment (phillipsdev) [inline ReadMe.md:47]: This is more of a reminder for me to go fix this, you shouldn't need the username, so that means I need to go update some code.

**Review by phillipsdev — APPROVED** [review]
First draft not bad 😄 I suspect we'll add more formatting, detail and such in the future.

**Review by emukupa — APPROVED** [review]
good start with the readme

**Pull request merged**